src/Hedgehog/Linq/Property.fs

```diff
@@ -6,39 +6,48 @@ open System
 open System.Runtime.CompilerServices
 open Hedgehog
 
+type PropertyThrows =
```

Comment:
Can't we just named this `Property` and play with attributes as in https://github.com/hedgehogqa/fsharp-hedgehog/blob/f0ea8a1a35eefd7ae0f5734fdef5eb1d3b75064e/src/Hedgehog/Seed.fs#L37?

Reply:
I was concerned that the names would collide in odd ways, but I was able to ride the line here and got the name `Property` to work. The API will contain:

- `Property`: Holds static factory functions (`ForAll`, `Success`, etc), and is the wrapper for `Property<unit>`.
- `PropertyExtensions`: Contains extension methods that target a `Property`/`Property<'a>`.
- `Property<'a>`: The type declared in `src/Hedgehog/Property.fs`.

Reply:
I like using `Property` in place of `Property<unit>` in the C# code.